cmd/microservice-ethereum-worker-server/main.go

```diff
@@ -77,9 +84,26 @@ func main() {
 		ethereumUrl              = util.PickEnvOrFatal(EnvEthereumHttpUrl)
 		networkId                = util.GetEnvOrFatal(EnvNetwork)
 
+		utilGaugesAddress        ethCommon.Address
```

Comment:
Break this up so it's more obvious that this is a new zero allocation instead of an assignment, and if I recall, for some reason the null address is not the zero value for ethCommon.Address (from memory!), so might be worth moving this closer to it's use and commenting the reason for why